spec/helpers/time_options_helper_spec.rb

```diff
@@ -42,6 +42,12 @@
     end
   end
 
+  describe "#format_full_date_and_time" do
+    it "formats the date and time correctly" do
+      expect(helper.format_full_date_and_time(Time.current.tomorrow.change(hour: 9))).to eq("Wednesday, 14 August 2019 at 9:00 am")
```

Comment:
I think this test will fail tomorrow when "Time.current.tomorrow" is Thursday.

Reply:
You could use Timecop to set "today" to be a specific date, and then time travel to it.

Reply:
Whoops. I wasn't thinking! This worked fine when testing `format_time` but of course only tests the time of day 🤦‍♂